### PR TITLE
docs: update quick start guide

### PR DESCRIPTION
**Pull request by leesj9476**
## Description

- Add: Quick Start Guide docs

closes: #XXXX

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by egonspace — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -31,4 +31,61 @@ To learn about Cosmos SDK, please refer [Cosmos SDK Docs](https://github.com/cos
...
+```
+sh init_node.sh sim
+```
+
+**Run**
+```
+simd start      # run a node
+```
+
+**visit with your browser**
```
Comment (0Tech) [inline README.md:52]: **Visit with your browser**

Comment (0Tech) [reply README.md:52]: minor typo? ;)

**Review by 0Tech — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (leesj9476) [reply README.md]: Edited. thx :)

**Review by 0Tech — APPROVED** [review]

**Review by 0Tech — APPROVED** [review]

**Pull request merged**